platform-configuration/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -32,11 +32,18 @@
 
     <reference-list id="initialManagedServiceList" interface="org.codice.ddf.configuration.ConfigurationWatcher"/>
 
-    <bean id="ConfigurationManager" class="org.codice.ddf.configuration.ConfigurationManager">
+    <bean id="ConfigurationManager" class="org.codice.ddf.configuration.ConfigurationManager" init-method="init">
         <cm:managed-properties persistent-id="ddf.platform.config"
                                update-strategy="component-managed" update-method="updated"/>
         <argument ref="initialManagedServiceList"/>
         <argument ref="configurationAdmin"/>
+        <property name="protocol" value="" />
```

Comment:
I don't think you want to set all of these things to empty strings at construction.  These should have some default values, or just not be included.